modules/serverless-ossm-setup-with-kourier.adoc

```diff
@@ -1,90 +0,0 @@
-// Module included in the following assemblies:
-//
-// * /serverless/integrations/serverless-ossm-setup.adoc
-
-:_content-type: PROCEDURE
-[id="serverless-ossm-setup-with-kourier_{context}"]
-= Integrating {SMProductShortName} with {ServerlessProductName} when Kourier is enabled
-
-You can use {SMProductShortName} with {ServerlessProductName} even if Kourier is already enabled. This procedure might be useful if you have already installed Knative Serving with Kourier enabled, but decide to add a {SMProductShortName} integration later.
-
-.Prerequisites
-
-* You have cluster administrator permissions on {ocp-product-title}, or you have cluster or dedicated administrator permissions on {rosa-product-title} or {dedicated-product-title}.
```

Comment:
We will drop this but with https://issues.redhat.com/browse/SRVKS-1114. So we need to deprecate that in 1.31 first, then drop it later.